### PR TITLE
[WIP] Nextron as a CLI

### PR DESCRIPTION
**Pull request by hadeeb**
These are some of ideas I had for `nextron` like #30, #33 ,etc.

I'll try to split this into smaller PR's.

Feedback are welcome.

### CODE REVIEW

File: dev.js [hunk 1]
```diff
@@ -18,18 +21,25 @@ async function dev() {
...
     cwd: __dirname,
-    stdio: 'inherit',
+    stdio,
   });
+
+  npxSync('yalc', ['publish'], { stdio });
+
   execSync('cd workspace');
-  execSync('yarn link nextron && yarn', {
+  npxSync('yalc', ['add', 'nextron'], {
```
Comment (saltyshiomix) [inline dev.js]: It's awesome :)
I added some scripts!

**Comment by hadeeb** [comment]
Python examples have an additional build step.
Any suggestion on how to include it in this workflow?

I was thinking something like a pre-package hook in `nextron.config.js`.

File: babel.js [hunk 2]
```diff
@@ -18,8 +18,8 @@ module.exports = (_context, opts = {}) => {
           },
         },
       ],
-      '@babel/typescript',
-      isTSEnabled && '@babel/preset-react',
+      '@babel/preset-react',
+      isTSEnabled && '@babel/typescript',
```
Comment (saltyshiomix) [inline babel.js]: 👍

File: bin/webpack/helpers.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ const getUserConfig = () => {
   }
 };
 
-const getwebpackConfig = (env) => {
+const getWebpackConfig = (env) => {
```
Comment (saltyshiomix) [inline bin/webpack/helpers.js]: 🙇

File: examples/custom-build-options/package.json [hunk 4]
```diff
@@ -13,22 +13,29 @@
...
     },
+    "files": [
+      {
+        "from": ".",
+        "filter": [
+          "package.json",
+          "app"
+        ]
+      }
+    ],
```
Comment (saltyshiomix) [inline examples/custom-build-options/package.json]: Good suggestions 👍

**Comment by saltyshiomix** [comment]
@hadeeb 

Ah, the button "Ready for review" is enough :)
I'll merge it to the master now.

**Pull request merged**